src/qt/optionsmodel.h

```diff
@@ -74,6 +74,7 @@ class OptionsModel : public QAbstractListModel
 
     /* Explicit setters */
     void SetPrune(bool prune, bool force = false);
+    void SetPrune(unsigned int prune_size_gb, bool force = false);
```

Comment:
In commit "gui: Fix prune settings after intro" (5fd852990cd0e5c94e384fcf6c5fb22891534f4f)

It seems dangerous to give two functions that change different settings ("bPrune" vs "nPruneSize") the exact same name distinguished only by a `bool` `int` overload. Should give the functions different names. Might suggest `SetPruneSizeGB` and `SetPruneEnabled`.

Reply:
Done.